scripts/generateAssetData/utils/portals.ts

```diff
@@ -1,31 +1,171 @@
 import type { ChainId } from '@shapeshiftoss/caip'
 import { ASSET_NAMESPACE, bscChainId, toAssetId } from '@shapeshiftoss/caip'
-import type { Asset } from '@shapeshiftoss/types'
+import type { Asset, AssetsByIdPartial } from '@shapeshiftoss/types'
+import { isSome } from '@shapeshiftoss/utils'
 import axios from 'axios'
 import qs from 'qs'
 import { getAddress, isAddressEqual, zeroAddress } from 'viem'
 import { CHAIN_ID_TO_PORTALS_NETWORK } from 'lib/market-service/portals/constants'
-import type { GetTokensResponse, TokenInfo } from 'lib/market-service/portals/types'
+import type {
+  GetPlatformsResponse,
+  GetTokensResponse,
+  PlatformsById,
+  TokenInfo,
+} from 'lib/market-service/portals/types'
 
+import generatedAssetData from '../../../src/lib/asset-service/service/generatedAssetData.json'
 import { colorMap } from '../colorMap'
 import { createThrottle } from '.'
 
+const assets = generatedAssetData as unknown as AssetsByIdPartial
+
+const PORTALS_API_KEY = process.env.REACT_APP_PORTALS_API_KEY
+
+export const getPortalTokens = async (nativeAsset: Asset): Promise<Asset[]> => {
+  const PORTALS_API_KEY = process.env.REACT_APP_PORTALS_API_KEY
+  if (!PORTALS_API_KEY) throw new Error('REACT_APP_PORTALS_API_KEY not set')
```

Comment:
missed here and let's toss the throw condition up on L23

Reply:
451f2e791c / f64ab7b219

Reply:
f64ab7b21901a51ab57f3241ae93180d21c0b259